tee-worker/omni-executor/aa-contracts/aa-demo-app/src/lib/constants.ts

```diff
@@ -10,138 +10,147 @@ const OmniAccountABI = (OmniAccountArtifact as any).abi || OmniAccountArtifact;
 const OmniAccountFactoryABI = (OmniAccountFactoryArtifact as any).abi || OmniAccountFactoryArtifact;
 const StandardERC20ABI = (StandardERC20Artifact as any).abi || StandardERC20Artifact;
 const SimplePaymasterABI = (SimplePaymasterArtifact as any).abi || SimplePaymasterArtifact;
+// TODO: Add actual ABI for ERC20PaymasterV1 when available
+const ERC20PaymasterV1ABI: any[] = [];
```

Comment:
Should this be resolved?

Reply:
I'll resolve this in another PR I'm about to open